packages/liquidator/test/index.js

```diff
@@ -20,7 +20,6 @@ const Poll = require("../index.js");
 let collateralToken;
 let syntheticToken;
 let financialContract;
-let uniswap;
```

Comment:
Removing reliance on Uniswap mock from these tests beacuse the actual pricefeed type doesn't matter. And now we have the `test` type (i.e. the PriceFeedMock) to use